cogs/strike.py

```diff
@@ -557,14 +548,18 @@ async def _confirm_manual_add_strike(self, strike_user: discord.User | discord.M
                     )
                     and interaction.channel == confirmation_message_channel
                     and "custom_id" in interaction.data
-                    and interaction.data["custom_id"] in {
+                    and interaction.data["custom_id"]
+                    in {
                         "yes_out_of_sync_ban_member",
                         "no_out_of_sync_ban_member",
                     }
                 ),
             )
 
-            if out_of_sync_ban_button_interaction.data["custom_id"] == "no_out_of_sync_ban_member":  # type: ignore[index, typeddict-item] # noqa: E501
+            if (
+                out_of_sync_ban_button_interaction.data["custom_id"]  # type: ignore[index, typeddict-item]
+                == "no_out_of_sync_ban_member"
+            ):
```

Comment:
See above comment about moving boolean expressions to separate flag rather than splitting lines